ribs/archives/_cvt_archive.py

```diff
@@ -259,13 +264,28 @@ def index_of(self, measures_batch):
         if self._use_kd_tree:
             _, indices = self._centroid_kd_tree.query(measures_batch)
             return indices.astype(np.int32)
-
-        # Brute force distance calculation -- start by taking the difference
-        # between each measure i and all the centroids.
-        distances = np.expand_dims(measures_batch, axis=1) - self.centroids
-
-        # Compute the total squared distance -- no need to compute actual
-        # distance with a sqrt.
-        distances = np.sum(np.square(distances), axis=2)
-
-        return np.argmin(distances, axis=1).astype(np.int32)
+        else:
+            expanded_measures = np.expand_dims(measures_batch, axis=1)
+            # Compute indices chunks at a time
+            if self._chunk_size is not None and \
+                    self._chunk_size < measures_batch.shape[0]:
+                indices = []
+                chunks = np.array_split(
+                    expanded_measures,
+                    np.ceil(len(expanded_measures) / self._chunk_size))
+                for _, chunk in enumerate(chunks):
+                    distances = chunk - self.centroids
+                    distances = np.sum(np.square(distances), axis=2)
+                    current_res = np.argmin(distances, axis=1).astype(np.int32)
+                    indices.append(current_res)
+                return np.concatenate(tuple(indices))
+            else:
+                # Brute force distance calculation --
```

Comment:
Format back to be wider

Reply:
Wider comments threw a pylint line-too-long error

Reply:
You'll need to format the comment so that it only has 80 chars per line.